app/assets/stylesheets/layout/_toolbar.sass

```diff
@@ -36,6 +36,11 @@
     font-weight: normal
     text-decoration: none
 
+@mixin toolbar-element-sizing
+  margin: 0.625rem 0 0
+  height: 40px
+  line-height: 22px
```

Comment:
If you omit the line-height, smaller icons not contained immediately in a toolbar item are no longer centered (this still applies to the clipboard symbol). For some other icons were this was a problem before, this no longer applies to due switching to flex.